libnmstate/schema.py

```diff
@@ -205,7 +205,23 @@ class OVSBridge(object):
     PORT_VLAN_MODE = 'vlan-mode'
     PORT_ACCESS_TAG = 'access-tag'
 
+    LA_SUBTREE = 'link-aggregation'
```

Comment:
`LA` is not a good abbreviation IMHO. Using the full name would match the other constants better.